source/includes/cloudstack/_s2s_vpns.md

```diff
@@ -0,0 +1,225 @@
+### Site-to-site VPN
+
+
+<!-------------------- LIST SITE-TO-SITE VPNS -------------------->
+
+#### List site-to-site VPNs
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/compute-on/test_area/sitetositevpns"
+
+# Example:
+```
+```json
+{
+  "data": [
+    {
+        "id": "d49b2922-0581-4587-94df-6fe719327d0f",
+        "name": "stargate",
+        "state": "Connected",
+        "vpcId": "3fe7d82a-f4c4-4552-ac3b-787fdafed4e7",
+        "gateway":"19.19.19.19",
+        "cidr":"10.12.0.2/22",
+        "ipSecPsk": "WtOBS9GRux2XtJPtHY2TUvrv",
+        "ikeEncryptionAlgorithm": "3des",
+        "ikeHashAlgorithm": "sha1",
+        "ikeDhGroup":"modp1536",
+        "ikeLifetime":86400,
+        "espEncryptionAlgorithm":"3des",
+        "espHashAlgorithm":"sha1",
+        "espPerfectForwardSecrecy":"modp1536",
+        "espLifetime":3600,
+        "dpd": false,
+        "forceEncap": false
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#service-connections">:service_code</a>/<a href="#environments">:environment_name</a>/sitetositevpns</code>
+
+Retrieve a list of all site-to-site VPNs in an [environment](#environments)
+
+Attributes | &nbsp;
+---------- | -----
+`id`<br/>*UUID* | The id of the site-to-site VPN
+`name`<br/>*string* | The name of the site-to-site VPN
+`state`<br/>*string* | The state of the site-to-site VPN. Can be Connected, Pending or Disconnected. If disconnected, you can try to use the [reset](#reset-a-site-to-site-vpn) operation
+`vpcId`<br/>*UUID* | The VPC for which the site-to-site VPN was created.
+`gateway`<br/>*string*  | The gateway of the network you want to connect to. NOTE: you cannot use a gateway that has already been used by a site-to-site VPN in your environment
+`cidr`<br/>*string*  | Cidr of the network you want to connect to.
+`ipSecPsk`<br/>*string*  | IPSec pre-shared key. Mmust contain at least 10 alphanumeric characters.
+`ikeEncryptionAlgorithm`<br/>*string*  | The Internet Key Exchange (IKE) policy for phase-1. The supported encryption algorithms are AES128, AES192, AES256, and 3DES.
+`ikeHashAlgorithm`<br/>*string*  | The IKE hash for phase-1. The supported hash algorithms are SHA1 and MD5.
+`ikeDhGroup`<br/>*string*  | A public-key cryptography protocol which allows two parties to establish a shared secret over an insecure communications channel. The supported options are None, Group-5 (1536-bit) and Group-2 (1024-bit).
```

Comment:
currently, _None_ maps to null in the UI, so it's not a valid value. We should say that it's optional, with only 2 valid values when sent.